contracts/extbridge/callback.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity ^0.4.24;
+
+contract Callback {
+    event RegisteredOffer(
+        address owner,
+        uint256 valueOrID,
+        address tokenAddress,
+        uint256 price
+    );
+
+    function RegisterOffer(
+        address _owner,
```

Comment:
Just a question:
Why do we need underscore here?

Reply:
I wrote like this caused by this reason (https://ethereum.stackexchange.com/questions/1351/what-is-the-purpose-of-underscores-in-front-of-parameters).

But It is not formal solidity guide. I will think about it again. And if we need to modify, I will apply the right style for all solidity code once. :)